src/frontend/src/pages/PPQPage.tsx

```diff
@@ -19,36 +19,36 @@ function PPQLandingPage() {
 
       <div className="  ppq-section">
         <div className="row what-is-a-ppq">
-        <div className='col-md-6'>
-          <h2 >PIA Pathway Questionnaire</h2>
-          <br />
-          <p className='col-mb-6'>
-            The PIA Pathways Questionnaire asks for some basic information about
-            the initiative you're assessing in the PIA. Your answers will help
-            us estimate{' '}
-          </p>
-         <ul className='col-mb-6 list'>
-          <li>The relative complexity of your PIA </li>
-          <li> Which PIA template you should fill out</li>
-          <li>Where to go for help with your PIA</li>
-          </ul> 
-          <div data-cy="ppq-btn" className="ctas">
-            <Link to="/ppq-form" className="btn-primary">
-              Get started{' '}
-              <FontAwesomeIcon className="icon" icon={faChevronRight} />
-            </Link>
+          <div className="col-md-6">
+            <h2>PIA Pathway Questionnaire</h2>
+            <br />
+            <p className="col-mb-6">
+              The PIA Pathways Questionnaire asks for some basic information
+              about the initiative you're assessing in the PIA. Your answers
+              will help us estimate{' '}
```

Comment:
empty space at the end. Don't see a purpose it's solving.

Reply:
I concur. We can--and should--be delegating spacing / layout to css.